handbook/ops/bizops/index.md

```diff
@@ -19,11 +19,26 @@ TODO
 
 This page describes Sourcegraph's analytics function, our data sources, and how to use our data tools.
 
-To reach us, mention @ericbm in the most relevant channel (i.e. #marketing for a marketing-related question).
+To reach us, mention @business-team in the most relevant channel (i.e. #marketing for a marketing-related question).
 
-## Submitting a data request
+## Active product analytics dashboards
 
-**Projects:** Proposals to capture and report on new data are in [GitHub issues in the Sourcegraph analytics repository](https://github.com/sourcegraph/analytics/issues). Provide the following data to help us understand and prioritize your request:
+Dashboards are structured by focus area or goal instead of team. As we have more and more projects being worked on and tracked concurrently, this makes it easier for the product, engineering and business teams to track, link to, and simplify the lifecycle of dashboards. For example, we don't need to think about when it's time to deprecate a look from the code intel dashboard. When the goal is removed, the dashboard is archived.
```

Comment:
"focus area or goal" is helpful, conceptually, but the text below makes it seem like it's actually just "goal"?

Is it correct to say that each goal listed on the team's "goals" page (e.g. https://about.sourcegraph.com/handbook/engineering/code-intelligence#goals) will have it's own dashboard?

If so, making that link explicit would be nice. If not, no need to adjust here.